### PR TITLE
Setup dependabot to update github-actions

### PR DESCRIPTION
**Pull request by nightlark**
### Summary

If merged this pull request will enable dependabot to open PRs for updating github-actions used by workflows in this repository.

### Proposed changes

- Add dependabot config file to enable github-actions update PRs

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
```
Comment (phlptp) [inline .github/dependabot.yml:4]: should this be limited to the .github directory or doesn't it matter much

**Comment by phlptp** [comment]
Is this automatic or is there a corresponding registration with a dependabot ci system?

File: .github/dependabot.yml (hunk 1 above)
Comment (nightlark) [reply .github/dependabot.yml:4]: It looks for a `.github/workflows` subdirectory in the path given

**Comment by nightlark** [comment]
Once the dependabot config file makes its way into `main` it will start running automatically (on Mondays).

**Review by phlptp — APPROVED** [review]

**Pull request merged**